config/settings.yml

```diff
@@ -87,6 +87,12 @@
   :ec2_network:
     :inventory_collections:
       :saver_strategy: batch
+  :eks:
+    :refresh_interval: 15.minutes
+    :streaming_refresh: false
```

Comment:
NOTE: disabled until I copy over the watch_thread inventory collector and persister classes.